src/main/java/com/courtcanva/ccfranchise/dtos/orders/OrderAcceptedAndCompletedGetDto.java

```diff
@@ -0,0 +1,21 @@
+package com.courtcanva.ccfranchise.dtos.orders;
+
+import com.courtcanva.ccfranchise.constants.OrderStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class OrderAcceptedAndCompletedGetDto {
```

Comment:
instead of create a new Dto, I sugguest reuse `OrderGetDto`, Iris and I used this one.
If  you find the fields you need are different from which in `OrderGetDto`, feel free to add fields you need. 
Contact me if you have questions.